pkg/rpcclient/unwrap/unwrap.go

```diff
@@ -25,12 +25,24 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/vm/vmstate"
 )
 
+// Exception is a type used for VM fault messages (aka exceptions). If any of
+// unwrapper functions encounters a FAULT VM state it creates an instance of
+// this type as an error using exception string. It can be used with [errors.As]
+// to get the exact message from VM and compare with known contract-specific
+// errors.
+type Exception string
```

Comment:
> as well as "FaultException" 

`unwrap.FaultException` seems to be more clear than `unwrap.Exception`, don't you think so?

Reply:
Now it's `unwrap.Exception`. I don't mind making it `unwrap.FaultException` if that's better. Most of `unwrap.X` things are to unwrap X from execution results. This one is different in that it's a type. We also have an `Err...` there. I'm open to suggestions of how to make it easier to understand that it's not a regular `unwrap.X` function.

Reply:
i would go just `Exception`

Reply:
Let's use it.